src/NzbDrone.Core/Download/Pending/PendingReleaseService.cs

```diff
@@ -97,7 +96,7 @@ public void AddMany(List<Tuple<DownloadDecision, PendingReleaseReason>> decision
 
                     var episodeIds = decision.RemoteEpisode.Episodes.Select(e => e.Id);
 
-                    var existingReports = episodeIds.SelectMany(v => alreadyPendingByEpisode[v] ?? Enumerable.Empty<PendingRelease>())
```

Comment:
`alreadyPendingByEpisode[v]` cannot be null as `ILookup` never return null. It returns an empty enumerable instead